src/g_doom/a_archvile.cpp

```diff
@@ -152,5 +154,6 @@ DEFINE_ACTION_FUNCTION_PARAMS(AActor, A_VileAttack)
 		P_RadiusAttack (fire, self, blastdmg, blastrad, dmgtype, 0);
 	}
 	if (!(target->flags7 & MF7_DONTTHRUST))
-		target->velz = Scale(thrust, 1000, target->Mass);
+		target->velz = Scale(thrust, 1000, target->Mass) * thrustmul;

```

Comment:
Yes. ThrustMul is merely a multiplier to how much the thrust being done is scaled by. Granted I could've been a bit more clever with the naming and just make it * thrustscale instead...